src/explore-education-statistics-admin/src/pages/release/data/components/DataFilesTable.module.scss

```diff
@@ -0,0 +1,36 @@
+@import '~govuk-frontend/dist/govuk/base';
+
+.table {
+  td {
+    vertical-align: middle;
+  }
+}
+
+.title {
+  max-width: 500px;
+}
+
+.fileSize {
+  width: 5rem;
+  text-align: right;
+}
+
+.fileStatus {
+  max-width: 126px;
+}
+
+.actions {
+  margin-bottom: 0;
+
+  @include govuk-media-query($until: desktop) {
+    align-items: start;
+    flex-direction: column;
+
+    a,
+    button {
+      margin-bottom: 1rem;
```

Comment:
Seems like the spacing is a bit excessive now:

![image](https://github.com/user-attachments/assets/88e522ee-49b8-47cf-8058-9d66e250caaa)

Additionally, we should be using `govuk-spacing` function for this.

1 spacing unit:

![image](https://github.com/user-attachments/assets/b99d2ce8-34f9-4a6c-9ebd-bc5d2ed081fe)

2 spacing units:

![image](https://github.com/user-attachments/assets/b746a11f-9f89-4a6e-99d9-0a6015c01f75)

Feels like 1 spacing unit might actually be fine